qt/aqt/clayout.py

```diff
@@ -791,7 +792,7 @@ def on_done(changes: OpChanges) -> None:
 
         update_notetype_legacy(parent=self.mw, notetype=self.model).success(
             on_done
-        ).run_in_background()
+        ).failure(lambda e: showWarning(str(e))).run_in_background()
```

Comment:
That's the default behaviour if you don't specify it, and the default uses the provided parent as well. :-)

Reply:
Right, I intended to do something more originally, like switching to the faulty template. But I don't see how that would be possible with the current implementation.